Frontend/src/helpers/RecipesDB_Simulator.tsx

```diff
@@ -0,0 +1,32 @@
+
+
+
+const recipeData = [
+    {
+        name: "Klassische Ramensuppe",
+        rating: "4.5",
+        preparationTime: "30min",
+        cookingTime: "15min",
+        difficulty: "Easy",
+        ingredients: "Liste der Zutaten",
+        instructions: "Kochanleitung"
+    },
+];
+
+
+
+const RecipesDB_Fetcher = () => {
+
+    // Lobic for fetching demanded recipes from database
+    // save them in recipes array
+
+    return 1;
+}
+
+
+export default recipeData;
+
+
+
+let one = RecipesDB_Fetcher();
+console.log(one);
```

Comment:
pls remove console.log()

Reply:
This command is only there to avoid an unused-variable-error of the one variable. Can be removed after integration of the component.